modules/cluster-logging-forwarding-separate-indices.adoc

```diff
@@ -0,0 +1,69 @@
+// Module is included in the following assemblies:
+//cluster-logging-external
+:_content-type: PROCEDURE
+[id="cluster-logging-forwarding-separate-indices_{context}"]
+= Forwarding JSON logs from containers in the same pod to separate indices
+
+[IMPORTANT]
+====
+JSON formatting of logs varies by application. Because creating too many indices impacts performance, limit your use of this feature to creating indices for logs that have incompatible JSON formats. Use queries to separate logs from different namespaces, or applications with compatible JSON formats.
```

Comment:
```suggestion
JSON formatting of logs varies by application. Because creating too many indices impacts performance, limit your use of this feature to create indices for logs with incompatible JSON formats. Use queries to separate logs from different namespaces, or applications with compatible JSON formats.
```
Do you mean "applications with **incompatible** JSON formats?"

Reply:
No, it is as its intended. If compatible JSON format, then query. If incompatible JSON formats, then indices.